### ISSUE TITLE
Support .cbz files

### ISSUE THREAD

**Issue by avidseeker** [oupson/jxlviewer#28]
.cbz file is a comic book file format that is simply a zip archive of images. Currently, there are no readers supporting the rendering of .jxl files within a .cbz archive.

**Comment by Quackdoc** [comment]
does tachiyomi not support it?

**Comment by AliceTreatPU** [comment]
Yes, Tachiyomi support it but the file (zip/rar/cbz/cbr) need to be stored and sorted according tachiyomi rule. Tachiyomi only open the file after the file is registered/scanned in Tachiyomi app. Tachiyomi can't open the file like external viewer and won't implement it (not planned).

For organized/sorted 100+ cbz file ts just inconvenience to use Tachiyomi as JPEGXL CBZ viewer.
